zypp/zyppng/media/medianetwork.h

```diff
@@ -45,8 +45,7 @@ class MediaHandlerNetwork : public zypp::media::MediaHandler
 protected:
   void attachTo(bool next) override;
   void releaseFrom(const std::string &ejectDev) override;
-  void getFile(const zypp::filesystem::Pathname &filename, const zypp::ByteCount &expectedFileSize_r) const override;
-  void getFiles(const std::vector<std::pair<zypp::filesystem::Pathname, zypp::ByteCount> > &files) const override;
+  void getFile ( const zypp::OnMediaLocation &file ) const override;
   void getDir(const zypp::filesystem::Pathname &dirname, bool recurse_r) const override;
```

Comment:
`find zypp/ -type f | xargs -d '\n' sed -i 's/zypp::filesystem::Pathname/Pathname/g'` maybe...

Reply:
That does not work in zyppng, different namespace. We could  pull Pathname into the zyppng namespace , but that would be a seperate commit and would touch a lot of unrelated code here